service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandler.java

```diff
@@ -127,8 +130,15 @@ public void sendNotificationsForOrder(final GeneratedOrderEvent orderEvent) {
 
         sendOrderNotificationToLocalAuthority(eventData.getCaseDetails(), eventData.getLocalAuthorityCode(),
             orderEvent.getMostRecentUploadedDocumentUrl());
-
         sendOrderIssuedNotificationToAdmin(eventData, orderEvent.getDocumentContents(), GENERATED_ORDER);
+
+        CaseData caseData = objectMapper.convertValue(eventData.getCaseDetails().getData(), CaseData.class);
+        List<Representative> representativesServedByEmail = representativeService.getRepresentativesByServedPreference(
+            caseData.getRepresentatives(), EMAIL);
+
+        if (!representativesServedByEmail.isEmpty()) {
```

Comment:
maybe we should  not check representativesServedByEmail.isEmpty() here. Now list of representatives is calculated twice, here and in private method (sendNotificationToRepresentatives). If implementation will change in one of them then it can produce unexpected result

Reply:
will change implementation of sendNotificationToRepresentatives to take in a list of representatives